### ISSUE TITLE
DiscordPresenceUpdateEventArgs user field is always null

### ISSUE THREAD

**Issue by lzrp** [Luigifan/DiscordSharp#3]
Hey,  you are missing a field assignment for the user field when creating DiscordPresenceUpdateEventArgs in file DiscordClient.cs at line 533+.

``` C#
 DiscordPresenceUpdateEventArgs dpuea = new DiscordPresenceUpdateEventArgs();
            dpuea.RawJson = message;
            var pserver = ServersList.Find(x => x.members.Find(y => y.user.id == message["d"]["id"].ToString()) != null);
            if (pserver != null)
            {
                var user = pserver.members.Find(x => x.user.id == message["d"]["id"].ToString());
```

You are missing
    dpuea.user = user;

**Comment by suicvne** [comment]
my bad, i will fix this and push a fix to nuget. thank you! :heart:

**Comment by suicvne** [comment]
Fixed in 19a578794281fb42dc8ed14ce49ce26b47942007
Also pushed to NuGet as `0.5.0.1-alpha`
:heart:
